src/reducetensor.cpp

```diff
@@ -43,6 +43,10 @@
 #include <iostream>
 #include <sstream>
 
+// headers from composable kernel, to get consistent ID mapping
+#include "../composable_kernel/composable_kernel/include/utility/data_type_enum.hpp"
+#include "../composable_kernel/composable_kernel/include/utility/reduction_enums.hpp"
```

Comment:
[Recommendation]
```suggestion
#include <../composable_kernel/composable_kernel/include/utility/data_type_enum.hpp>
#include <../composable_kernel/composable_kernel/include/utility/reduction_enums.hpp>
```
if this works.

Reply:
Yes.  Thanks.   Should be consistent with the using of other includings.